lib/uplinkc/testdata/access_scope_test.c

```diff
@@ -25,7 +25,31 @@ int main(int argc, char *argv[])
         requiref(strcmp(scopeSerialized, scopeStr) == 0,
                  "got invalid serialized %s expected %s\n", scopeSerialized, scopeStr);
 
+        char *satelliteAddres = get_scope_satellite_address(scope, err);
+        require_noerror(*err);
```

Comment:
Let's add a check that the `satelliteAddress` is not nil and it isn't an empty string.

Reply:
done